cparse.nim

```diff
@@ -2310,8 +2310,18 @@ proc parseTemplate(p: var Parser): PNode =
       if p.tok.xkind != pxAngleRi:
         while true:
           if p.tok.xkind == pxSymbol and
-              (p.tok.s == "class" or p.tok.s == "typename"): getTok(p)
-          result.add skipIdent(p, skType)
+              (p.tok.s == "class" or p.tok.s == "typename"): 
+                      getTok(p)
+                      var identDefs = newNodeP(nkIdentDefs, p)
+                      identDefs.addSon(skipIdent(p, skType), ast.emptyNode, ast.emptyNode)
+                      result.add identDefs
+          if p.tok.xkind == pxSymbol and isIntType(p.tok.s) and
+              p.tok.s != "double" and p.tok.s != "float":
+                      var staticTy = newNodeP(nkStaticTy, p)
```

Comment:
Please use the same indentation rules like the rest of the file.

Reply:
Oups sorry correcting that.